classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @id = label
+    @student = []
+  end
+
+  def add_student(student)
+    @student.push(student)
```

Comment:
- [ ] Please add Getters for @stusents (attrb_reader) to serve both ways relationships between classroom and student classes. and please make it students not student.